index.html

```diff
@@ -77,7 +69,47 @@ <h4 class="sub-title">I am a Software Developer</p>
 
       </section>
 
+      <section class="wrap">
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        
+      <header>
+        <h1 class="heading">Multi-Purpose Stories</h1>
+     </header>  
+     
+     <main>
+    <section>
+        <div>
+    <p class="text">
+        A daily selection of privately 
+         personalized reads; no accounts or 
+          sign-ups required. has been the 
+           industry's standard dummy text ever 
+           since the 1500s, when an unknown 
+           printer took a standard dummy text.
+    </p>
+    </div>
+    
+    <nav class="button">
+        <button class="button1" type="css">css</button>
+        <button class="button2" type="html">html</button>
+        <button class="button3" type="bootstrap">bootstrap</button>
+        <button class="button4" type="Ruby">Ruby</button>
+        <button class="button5" type="See Project">See Project</button>
+    </nav>
+    
+    </section>
+    </section>
+
     </main>
+
    
 </body>
 </html>
```

Comment:
- [ ] As in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you were expected to implement two sections: **The projects** section which you implemented ✔ and the **About myself** section which your project is missing ⛔. Please go ahead and add the missing section to your project 📝.

**About myself section:**
![image](https://user-images.githubusercontent.com/43172164/198828296-bff1f8a4-f9ba-4f2a-8043-8de92f57ad95.png)